wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/spi/WorkspaceDao.java

```diff
@@ -72,7 +73,7 @@ WorkspaceImpl update(WorkspaceImpl update)
    * @throws NullPointerException when {@code id} is null
    * @throws ServerException when any other error occurs during workspace removing
    */
-  void remove(String id) throws ServerException;
+  Optional<WorkspaceImpl> remove(String id) throws ServerException;
```

Comment:
Please update java doc according to made changes.

Reply:
updated